lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -649,6 +705,27 @@ protected function handle_font_file_upload( $file ) {
 		return $uploaded_file;
 	}
 
+	/**
+	 * Handles file upload error
```

Comment:
```suggestion
	 * Handles file type upload error.
```

Reply:
Actually this function does handle all upload errors. But I did forget the period--will add that!